tests/Client/UpdaterTest.php

```diff
@@ -97,123 +125,257 @@ public function providerRefreshRepository()
         return $this->getKeyedArray([
             [
                 'delegated',
-                [
-                    'root' => 3,
-                    'timestamp' => 3,
-                    'snapshot' => 3,
-                ],
                 [
                     'root' => 5,
                     'timestamp' => 5,
                     'snapshot' => 5,
+                    'targets' => 5,
                 ],
             ],
             [
                 'simple',
-                [
-                    'root' => 2,
-                    'timestamp' => 2,
-                    'snapshot' => 2,
-                ],
                 [
                     'root' => 3,
                     'timestamp' => 3,
                     'snapshot' => 3,
+                    'targets' => 3,
                 ],
             ],
         ], 0);
     }
 
     /**
-     * Asserts that files in the repo are at expected versions.
+     * Asserts that files in the client repo are at expected versions.
      *
      * @param array $expectedVersions
      *   The expected versions.
      *
      * @return void
      */
-    protected function assertRepoVersions(array $expectedVersions): void
+    protected function assertClientRepoVersions(array $expectedVersions): void
```

Comment:
Updated the name and comment here to make it clear we are asserting the client repo state